content/cloud/pricing-and-billing.md

```diff
@@ -52,9 +52,9 @@ Fermyon charges the tax mandated by the local taxing authority. Your bill will i
 
 No, at this time Fermyon Cloud only accepts reoccurring payments. If at any point you decide you no longer want to continue with your paid plan, you are eligible to cancel your subscription. 
 
-**What happens if there's an issue with my payment method during a reoccurring charge?**
+**What happens if there's an issue with my payment method **during a **recurring** charge?****
```

Comment:
The nested bolds don't do anything.  Not sure what you're aiming for with the emphasis-upon-emphasis?  The original (with your wording tweak) seems fine to me

```suggestion
**What happens if there's an issue with my payment method during a recurring charge?**
```